querybook/server/logic/data_element.py

```diff
@@ -75,7 +96,11 @@ def create_data_element_association(
     primitive_type: str = None,
     session=None,
 ):
-    if data_element_tuple is None:
+    if (
+        data_element_tuple is None
+        or not data_element_tuple.name
```

Comment:
nit: how about if not data_element_tuple or not data_element_tuple.name ...
so its more consistent to read